instance/migrations/0140_auto_20210315_1246.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.12 on 2021-03-15 12:46
```

Comment:
@0x29a: nit: use a proper name for the migration file that explains what the change is.